scripts/deployment/l2/deploy_04_gnosis_safe_same_address_multisig.js

```diff
@@ -0,0 +1,56 @@
+/*global process*/
+
+const { ethers } = require("hardhat");
+const { LedgerSigner } = require("@anders-t/ethers-ledger");
+
+async function main() {
+    const fs = require("fs");
+    const globalsFile = "globals.json";
+    const dataFromJSON = fs.readFileSync(globalsFile, "utf8");
+    let parsedData = JSON.parse(dataFromJSON);
+    const useLedger = parsedData.useLedger;
+    const derivationPath = parsedData.derivationPath;
+    const providerName = parsedData.providerName;
+    let EOA;
+
+    const provider = await ethers.providers.getDefaultProvider(providerName);
+    const signers = await ethers.getSigners();
+
+    if (useLedger) {
+        EOA = new LedgerSigner(provider, derivationPath);
+    } else {
+        EOA = signers[0];
+    }
+    // EOA address
+    const deployer = await EOA.getAddress();
+    console.log("EOA is:", deployer);
+
+    // Transaction signing and execution
+    console.log("4. EOA to deploy GnosisSafeMultisig");
```

Comment:
```suggestion
    console.log("4. EOA to deploy GnosisSafeSameAddressMultisig");
```

Reply:
No need, the output contract name and the ultimate address name is what matters.

Reply:
Nevertheless, I already reviewed the PR so I do not want to add additional work to you this time. But please, consider that this would be helpful next time